modules/core/src/test/java/org/apache/ignite/internal/testframework/IgniteTestUtils.java

```diff
@@ -745,4 +746,12 @@ public static <T> T await(CompletionStage<T> stage) {
 
         return null;
     }
+
+    /**
+     * Returns a consumer that does nothing.
+     */
+    public static <T> Consumer<T> doNothingConsumer() {
```

Comment:
Why is this method needed? Isn't `t -> {}` obvious by itself? This class is huge enough already)

Reply:
I'm not able to format the code with `t -> {}` so that it is on the same line, formatting will definitely be in order with this approach, let's leave it as it is.

Reply:
Thanks, I'll fix it.